docs/experience_date_search/index.md

```diff
@@ -10,7 +10,7 @@ has_toc: false
 
 Experience date search enables you to filter a Creator's experiences with a separate result for each departure date of each experience.
 
-The experience date search is executed using the [experience_date_search tag]({% link docs/reference/tags/experience_date_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object and a `search` [Search]({% link docs/reference/objects/search_query.md %}) object.
+The experience date search is executed using the [experience_date_search tag]({% link docs/reference/tags/experience_date_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object, a `search` [Search]({% link docs/reference/objects/search_query.md %}) object, and an `available_months` array of Date objects (see [Liquid documentation for Dates](https://shopify.github.io/liquid/filters/date).
```

Comment:
```suggestion
The experience date search is executed using the [experience_date_search tag]({% link docs/reference/tags/experience_date_search_tag/index.md %}). The tag returns an `items` array of [ExperienceDates]({% link docs/reference/objects/product/experience_date.md %}), a `paginate` [Pagination]({% link docs/reference/objects/pagination.md %}) object, a `search` [Search]({% link docs/reference/objects/search_query.md %}) object, and an `available_months` array of `Date` objects (see [Liquid documentation for Dates](https://shopify.github.io/liquid/filters/date).
```

formatting: adjusting this to be formatted consistently with the other instances on the page